docs/howto/webhook_receiver.md

```diff
@@ -0,0 +1,335 @@
+# How to write a webhook receiver
+
+This guide shows how to add a receiver using the webhook technique.
+
```

Comment:
Could you provide a quick overview of the steps the reader will take in writing a webhook receiver? I think that would help set expectations.

Reply:
Sure! Thanks for the suggestion.